Source/AlertController.swift

```diff
@@ -353,7 +353,7 @@ public class AlertController: UIViewController {
     }
 
     private func addChromeTapHandlerIfNecessary() {
-        if self.behaviors.contains(.dismissOnOutsideTap) {
+        guard self.behaviors.contains(.dismissOnOutsideTap) else {
```

Comment:
Could you change this to be `if !`? I prefer using `guard` only in combination with unwrapping an optional.